config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
   root "pages#home"
   get "about", to: "pages#about"
+  resources :articles, only: [:show]
```

Comment:
Create routes for `articles`, by default there are 7 routes:
![image](https://user-images.githubusercontent.com/60863955/157408008-ffaab35b-bd12-455e-aaf0-872164080349.png)
and we only choose to write the `show` action